templates/tom_targets/partials/galaxy_table.html

```diff
@@ -18,12 +18,22 @@
       <td>{{ galaxy.PCC|floatformat:3 }}</td>
       <td>{{ galaxy.Offset|floatformat:2 }}</td>
       {% if galaxy.Dist and galaxy.Dist|floatformat != 'nan' %}
-        <td>{{ galaxy.Dist|floatformat:1 }}&nbsp;&plusmn;&nbsp;{{ galaxy.DistErr|floatformat:1 }} </td>
+        {% if galaxy.DistErr.0 == galaxy.DistErr.1 or galaxy.DistErr.1|floatformat == 'nan' %}
+          <td>{{ galaxy.Dist|floatformat:1 }}&nbsp;&plusmn;&nbsp;{{ galaxy.DistErr.0|floatformat:1 }} </td>
+        {% else %}
+          <td style='min-width:8em;'>
+	    {{ galaxy.Dist|floatformat:1 }}<span class="supsubdist"><sub>-{{ galaxy.DistErr.1|floatformat:1 }}</sub><sup>+{{ galaxy.DistErr.0|floatformat:1 }}</sup></span>
```

Comment:
I think @dsand is right, you have the upper and lower error bars flipped here (and in line 35).

Reply:
Yes he was correct! The newest commit fixed this issue!

Reply:
@noahfranz13 I am sure you just did that to make it seem like I contribute to this project!
Thanks!